tests/chainerx_tests/unit_tests/routines_tests/test_indexing.py

```diff
@@ -254,6 +254,21 @@ def forward_xp(self, inputs, xp):
         return b,
 
 
+@pytest.mark.parametrize_device(['native:0', 'cuda:0'])
+@pytest.mark.parametrize('shape,indices,axis', [
+    # Invalid: Index out of bounds
+    ((2, 3), [2], 0),
+    ((2, 3), [-3], 0),
+])
+def test_take_index_error(device, shape, indices, axis):
+    a = array_utils.create_dummy_ndarray(chainerx, shape, 'float32')
+    indices = numpy.array(indices).astype(numpy.int32)
+    if device.backend.name == 'cuda':
+        pytest.skip()
```

Comment:
How about passing a skip body something along.
```suggestion
        pytest.skip('take with cuda backend does not raise for index error')
```

Reply:
Done 👍